web-registry/src/components/templates/ProjectDetails/ProjectDetails.utils.tsx

```diff
@@ -147,12 +152,20 @@ export function getParty(
   if (!party) {
     return undefined;
   }
+  let image: string;
+  if (!!party.image) {
+    image = party.image;
+  } else if (!party.image && party.type === 'USER') {
+    image = DEFAULT_PROFILE_USER_AVATAR;
+  } else {
+    image = DEFAULT_PROFILE_COMPANY_AVATAR;
+  }
```

Comment:
```suggestion
  const image = party.image
    ? party.image
    : getDefaultAvatar(party)
```

using `getDefaultAvatar` fn from https://github.com/regen-network/regen-web/pull/1896/files#diff-f2e74744965201930f8dce1db86db894175b09461325109643b09a0a2989fa18

Reply:
i think i'll just mark this as a TODO for now since that code above isn't merged yet
and we can adjust it once it's merged